src/Microsoft.AspNetCore.Authentication.AzureAD.UI/AzureADAuthenticationBuilderExtensions.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.using Microsoft.AspNetCore.Authorization;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Authentication.AzureAD.UI;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    /// <summary>
+    /// Extension methods to add Azure Active Directory Authentication to your application.
+    /// </summary>
+    public static class AzureADAuthenticationBuilderExtensions
+    {
+        /// <summary>
+        /// Adds JWT Bearer authentication to your app for Azure Active Directory B2C Applications.
```

Comment:
Still says B2C in lots of comments, variables, etc. in this PR. Please check every file.

Reply:
Yeah, I ran `Get-ChildItem .\src\Microsoft.AspNetCore.Authentication.AzureAD.UI\, .\test\Microsoft.AspNetCore.Authentication.AzureAD.UI.Test\ -File -Recurse | select-string B2C` and it was just this file and another comment in another file that I didn't catch.